README.md

```diff
@@ -57,33 +54,29 @@ Use the following URI for GET and DELETE operation on the Restore Point resource
 https://management.azure.com/subscriptions/{SubscriptionID}/resourceGroups/{ResourceGroupName}/providers/Microsoft.Compute/restorePointCollections/{RestorePointCollectionName}/RestorePoints/{RestorePointName}?api-version=2021-03-01
  
 For update of an existing Restore Point Collection resource, only update of tags is permitted.
-## Create Disk
 
+## Create Disk
 1. Get the Disk Restore ID by either using GET call on Restore Point Collection and expanding Restore Points or by doing a call on Restore Point(s). 
-2. You can then use this ARM template to create a disk using diskRestorePoint: https://github.com/Azure/Virtual-Machine-Restore-Points/blob/main/createDiskFromDiskRestorePoint.json
-
-Alternatively, you can also use the below Rest API call (same as https://docs.microsoft.com/en-us/rest/api/compute/disks/createorupdate):
+2. You can use the below Rest API call to create a disk using diskRestorePoint. [API documentation.](https://docs.microsoft.com/en-us/rest/api/compute/disks/createorupdate)
 
 PUT https://management.azure.com/subscriptions/49151e6d-fa6e-4985-ae85-00548ec78853/resourceGroups/dfgdf_group/providers/Microsoft.Compute/disks/restore_disk1?api-version=2020-12-01
 
 Request Body:
 {
-"location": "East US",
-"properties": {
-"creationData": {
-"createOption": "Restore",
-"sourceResourceId": "/subscriptions/{SubscriptionID}/resourceGroups/{ResourceGroupName}/providers/Microsoft.Compute/restorePointCollections/{RestorePointCollectionName}/restorePoints/{RestorePointName}/diskRestorePoints/{DiskRestorePointName}"
-}
-}
+   "location": "East US",
+   "properties": {
+      "creationData": {
+      "createOption": "Restore",
+      "sourceResourceId": "/subscriptions/{SubscriptionID}/resourceGroups/{ResourceGroupName}/providers/Microsoft.Compute/restorePointCollections/{RestorePointCollectionName}/restorePoints/{RestorePointName}/diskRestorePoints/{DiskRestorePointName}"
+      }
+   }
 }
 
 ## SAS using Disk Restore Points
-
 We are providing a BeginGetAccess API through which the user can directly pass the ID of the diskRestorePoint to create a SAS to the underlying disk. 
 
-   If there is no active SAS on the incremental snapshot of the restore point, then a new SAS will be created and Url returned to the user. 
-
-   If there is already an active SAS, the SAS will be extended and original SASUrl will be returned. 
+* If there is no active SAS on the incremental snapshot of the restore point, then a new SAS will be created and Url returned to the user. 
+* If there is already an active SAS, the SAS will be extended and original SASUrl will be returned. 
 
 POST  
 
```

Comment:
Simply name the .docx files now "Api Documentation.docx" since they are already within the respective folders.

Reply:
Done